### ISSUE TITLE
edits to Markov asset pricing lecture

### ISSUE THREAD

**Issue by jstac** [QuantEcon/lecture-jax#95]
Do we need `static_argnums` in 

```
power_iteration_sr = jax.jit(power_iteration_sr, static_argnums=(1,2))
```

If not let's cut.

**Comment by HumphreyYang** [comment]
Hi @jstac,

This is not necessary in this lecture, as we only use it with the default values.

However, we might want to be slightly careful here. If readers hope to increase the number of iterations or the seed using this function, they will run into `TracerIntegerConversionError`. 

Should we keep it the same to remove the `static_argnums`?

**Comment by jstac** [comment]
Thanks @HumphreyYang .

We explain `static_argnums` after it's used to JIT this function.  So let's just get rid of it.
